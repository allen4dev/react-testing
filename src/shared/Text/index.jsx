import React from 'react';

const Text = () => {
  return <p className="Text">Dummie text to see snapshot testing</p>;
};

export default Text;
