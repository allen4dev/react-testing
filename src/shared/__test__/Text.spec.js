import React from 'react';
import { shallow } from 'enzyme';

import Text from './../Text';

describe('<Text />', () => {
  it('should render Text component', () => {
    const text = <Text />;

    expect(text).toMatchSnapshot();
  });
});
