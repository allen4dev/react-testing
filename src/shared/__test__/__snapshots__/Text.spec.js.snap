// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Text /> should render Text component 1`] = `<Text />`;
